src/client/screens/Feed/index.js

```diff
@@ -201,7 +208,9 @@ class FeedPage extends React.Component {
         onShareClick={() => {
           this.share(event, item);
         }}
-        isSharing={this.state.isSharing} />
+        isSharing={this.state.isSharing}
+        isLiked={this.checkLiked(item)}
```

Comment:
do you use this isLiked property? If not, please remove it and also the called function